jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationStackNodeTest.kt

```diff
@@ -48,31 +48,31 @@ class CloudFormationStackNodeTest {
     @Before
     fun setup() {
         whenever(mockCfnClient.describeStackResources(any<DescribeStackResourcesRequest>())).thenReturn(
-            DescribeStackResourcesResponse.builder()
-                .stackResources(
-                    StackResource.builder().resourceType(LAMBDA_FUNCTION_TYPE).resourceStatus(ResourceStatus.CREATE_COMPLETE).logicalResourceId("processor").build(),
-                    StackResource.builder().resourceType(LAMBDA_FUNCTION_TYPE).resourceStatus(ResourceStatus.CREATE_COMPLETE).logicalResourceId("processor2").build(),
-                    StackResource.builder().resourceType("a dynamodb table").resourceStatus(ResourceStatus.CREATE_COMPLETE).build(),
-                    StackResource.builder().resourceType("an IAM role").resourceStatus(ResourceStatus.CREATE_COMPLETE).build()
-                )
-                .build()
+                DescribeStackResourcesResponse.builder()
```

Comment:
Why are you changing this formatting?

Reply:
I've not changed the formatting, I just touched this file and the IDE reformatted it itself. I am using the default Kotlin code style settings (indent size 4, continuation indent 8) and I've never changed them.

Reply:
There are 2 Kotlin code styles in Intellij, make sure you aren't using the obsolete one.

Reply:
May be, they changed it in a recent update in IntelliJ IDEA. The parenthesis '(' is followed by a continuation indent.

Reply:
So, I changed Kotlin code style in my IDE: Editor > Code Style > Kotlin > Set from... > Predefined Style > Kotlin style guide. Now, it seems to match the style of our codebase...hopefully.